src/address/address.go

```diff
@@ -64,7 +64,7 @@ func AddrForKey(publicKey ed25519.PublicKey) *Address {
 		buf[idx] = ^buf[idx]
 	}
 	var addr Address
-	var temp []byte
+	var temp = make([]byte, 0, 128)
```

Comment:
There are better ways to implement this function if we really cared about performance. I guess it doesn't hurt anything to keep the current implementation and preallocate the slice, but I think this should be 32 bytes (since temp is at most the length of the public key, which is 32 bytes).

Reply:
True; I miscounted that for whatever reason thinking those were bits, and not bytes. :D